src/cgame/cg_main.cpp

```diff
@@ -998,6 +1000,8 @@ cvarTable_t cvarTable[] =
 
 	{ &etj_projection, "etj_projection", "0", CVAR_ARCHIVE },
 	{ &etj_saveMsg, "etj_saveMsg", "^7Saved", CVAR_ARCHIVE },
+
+	{ &etj_FPSMeterUpdateInterval, "etj_FPSMeterUpdateInterval", "250", CVAR_ARCHIVE },
```

Comment:
I'm not sure about the default value on this tbh, I would make this 8 t match the original default rate at 125FPS.